integration_tests/tests/cucumber.rs

```diff
@@ -98,8 +99,8 @@ use crate::utils::{
 pub const LOG_TARGET: &str = "cucumber";
 pub const LOG_TARGET_STDOUT: &str = "stdout";
 const CONFIRMATION_PERIOD: u64 = 4;
-const NUM_RETIRES: u64 = 240;
-const RETRY_TIME_IN_MS: u64 = 500;
+const TWO_MINUTES_WITH_HALF_SECOND_SLEEP: u64 = 240;
```

Comment:
This name is confusing. It's not actually used in sleep. It's kind of a number of retries, but it's multiplied by some values in the code. I suppose it is a value when the number of retries runs only this many times with the half second sleep it will be two minutes. Right?

Reply:
DM suggested it made more sense in the end so we'll leave it